vpoly.s

```diff
@@ -262,7 +262,7 @@ continue:
 
 	# a5 is vlmax-1 for e32m1
 	li t0, -1
-	vsetvli a5, t0, e32, m1, ta, ma
+	vsetvli a5, t0, e32, m1, ta, mu
```

Comment:
Only the precomputation loop uses masked instructions, so I think this is the only vsetvli instruction that needs to change.

Reply:
ah, true I changed the others to ma/ta.
I suppose one could also wrap just the precomp block in ta,mu and use ta,ma directly after it ended, but since that's all setup anyways, I didn't add it. (I also don't know if that would even be faster)